NOTICE.txt

```diff
@@ -8,7 +8,9 @@ This document includes a list of open source components used in Mattermost Serve
 
 ---
 
-This product contains a modified portion of 'jquery-dragster', a drag and drop listener
+## jquery-dragster
```

Comment:
This isn't a golang dependency. This is for the webapp. Maybe we should review the list and match what I posted?